internal/resource/resource.go

```diff
@@ -23,13 +23,26 @@ import (
 )
 
 // AnyResourceV1 takes a BindPlane resource name, kind, type, parameters
-// and returns a bindplane.observiq.com/v1.AnyResource. Supported resources
-// are Sources, Destinations, and Processors. For configurations, use
-// AnyResourceFromConfigurationV1.
-func AnyResourceV1(rName, rType string, rKind model.Kind, rParameters []model.Parameter) (model.AnyResource, error) {
```

Comment:
AnyResourceV1 was updated to take optional rProcessors as []model.ResourceConfiguration. This way, Processor bundles can be converted to AnyResource, just like Source, Processor, Destination, and Extension. Internally, Processor Bundles are just Processors, but they utulize the embedded Processors field.